rules.md

```diff
@@ -58,8 +56,6 @@ Response bodies no matter the response must use ref for their schemas.</details>
 When responding with a 4xx or 5xx, you must add and use the approved error response in the response schema</details>
 <details><summary>path-responses-must-use-qualified-status-codes (error)</summary>
 You must only use allowed http response codes</details>
-<details><summary>path-schema-declarations-must-define-examples (error)</summary>
-Every schema declaration must define at least one example.</details>
```

Comment:
We are removing these? Just confirming

Reply:
If this is intentional, can you provide a brief explanation why.

Are there unit tests for these rules that need removed as well?

Reply:
This rule ends up being obsolete with component-parameters-must-include-examples. Since we are requiring path schemas to have $ref, the examples get picked up from the components.parameters examples. I can leave it in as an extra check, but spectral ends up looking at the examples from the components.parameters schema anyways instead of the jsonPath given in the actual rule being removed. I will remove this unit test, I rushed past that and forgot. Sorry about that.